src/commands/dockerInstaller.ts

```diff
@@ -84,3 +79,26 @@ export class MacDockerInstaller extends DockerInstallerBase {
         terminal.show();
     }
 }
+
+export async function showDockerInstallNotification(): Promise<void> {
+    const dockerInstallMessage = ext.os.platform === 'linux'
+        ? localize('vscode-docker.commands.dockerInstaller.installDockerInfo', 'Docker is not installed. Would you like to open a page with more information?')
+        : localize('vscode-docker.commands.dockerInstaller.installDocker', 'Docker Desktop is not installed. Would you like to Install?');
+
+    const confirmationPrompt: vscode.MessageItem = ext.os.platform === 'linux' ? { title: 'Open' } : { title: 'Install' };
```

Comment:
Are `'freebsd'` and `'openbsd'` valid options? They would be excluded by this `=== 'linux'`.